roles/installer/templates/deployments/deployment.yaml.j2

```diff
@@ -53,7 +53,7 @@ spec:
               mkdir -p /etc/pki/ca-trust/extracted/{java,pem,openssl,edk2}
               update-ca-trust
 {% endif %}
-{% if projects_persistence|bool %}
+{% if projects_persistence|bool and is_k8s|bool %}
```

Comment:
I'm a bit confused by this modification. Does that mean we can't use projects_persistence with OpenShift ?

Reply:
@dsavineau These permissions changes are needed when using NFS PVC's for projects_persistence.  We cannot do this in Openshift because of security restrictions (specifically the fsGroup below).  Setting them only for k8s deployments is a hold-over until we find a better solution for Openshift.